### PR TITLE
Adding export

### PR DESCRIPTION
**Pull request by InnaMel**

### CODE REVIEW

File: DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs [hunk 1]
```diff
@@ -1,15 +1,33 @@
...
+using System.Windows;
 
 namespace DuplicatesFinder_v4.ViewModels
 {
-    public class DuplicatesViewModel
+    public class DuplicatesViewModel : INotifyPropertyChanged
     {
+        public event PropertyChangedEventHandler PropertyChanged;
+
+        public string pathWithAppFolder;
```
Comment (YuriZakitny) [inline DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs]: make as static property

File: DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs [hunk 2]
```diff
@@ -19,23 +37,46 @@ public void Divide(ObservableCollection<ObservableCollection<FileConsist>> whole
...
+                    CollectionForDuplicatesView.Add(eachDuplicatesForView);
+                }
+            }
+        }
+
+        public async void SaveToTxt()
+        {
+            var dateCreation = DateTime.Now.ToShortDateString();
+            var getCurrentNameUser = Environment.UserName;
+            pathWithAppFolder = Path.Combine($"C:\\Users\\{getCurrentNameUser}\\Downloads", "DuplicatesFinder");
```
Comment (YuriZakitny) [inline DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs]: init just once

File: DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs [hunk 3]
```diff
@@ -19,23 +37,46 @@ public void Divide(ObservableCollection<ObservableCollection<FileConsist>> whole
...
+            var setFileSaveName = $"Dublicates_{dateCreation}.txt";
+
+            Directory.CreateDirectory(pathWithAppFolder);
+
+            var setFilePath = $"{pathWithAppFolder}\\{setFileSaveName}";
+            var option = new JsonSerializerOptions { WriteIndented = true };
```
Comment (YuriZakitny) [inline DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs:75]: use Path.Combine

File: DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs [hunk 4]
```diff
@@ -19,23 +37,46 @@ public void Divide(ObservableCollection<ObservableCollection<FileConsist>> whole
...
+
+            if (File.Exists(setFilePath))
+            {
+                File.Delete(setFilePath);
+            }
+
+            using (FileStream exportFile = new FileStream(setFilePath, FileMode.OpenOrCreate))
```
Comment (YuriZakitny) [inline DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs:82]: var

File: DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs [hunk 5]
```diff
@@ -19,23 +37,46 @@ public void Divide(ObservableCollection<ObservableCollection<FileConsist>> whole
...
-                            FilePath = fileConsist.FilePath, 
-                            DateTimeCreateString = (fileConsist.DateTimeCreate).ToString("g"), 
-                            SizeFile = Math.Round((fileConsist.SizeFile / 1000000), 3), 
-                            ExtensionFile = (fileConsist.ExtensionFile).Remove(0,1).ToUpper() };
-                        eachDuplicatesForView.FullInfoFiles.Add(fileConsistMedium);
+                        await JsonSerializer.SerializeAsync<FileConsist>(exportFile, item, option);
                     }
-                    CollectionForDuplicatesView.Add(eachDuplicatesForView);
                 }
+                exportFile.Close();
```
Comment (YuriZakitny) [inline DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs]: no need because in using

File: DuplicatesFinder_v4/ViewModels/MainViewModel.cs [hunk 6]
```diff
@@ -131,42 +127,101 @@ public ICommand OnClickSearch
                 {
                     DuplicatesViewModel.CollectionForDuplicatesView.Clear();
 
-                    if (ispics == false && isdocs == false && isvideos == false )
+                    if (ispics == false && isdocs == false && isvideos == false)
```
Comment (YuriZakitny) [inline DuplicatesFinder_v4/ViewModels/MainViewModel.cs:177]: make PropertyChanged as property
order properties and methods

**Review by YuriZakitny — COMMENTED** [review]
done

File: DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs (hunk 2 above)
Comment (InnaMel) [reply DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs]: has done

File: DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs (hunk 3 above)
Comment (InnaMel) [reply DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs:75]: fixed

File: DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs (hunk 4 above)
Comment (InnaMel) [reply DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs:82]: fixed

File: DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs (hunk 5 above)
Comment (InnaMel) [reply DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs]: ok, tnx

File: DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs (hunk 1 above)
Comment (InnaMel) [reply DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs]: has done

File: DuplicatesFinder_v4/ViewModels/MainViewModel.cs (hunk 6 above)
Comment (InnaMel) [reply DuplicatesFinder_v4/ViewModels/MainViewModel.cs:177]: has done

**Pull request merged**